src/algorithms/acquisition/adapters/beidou_b2ad_pcps_acquisition.h

```diff
@@ -0,0 +1,172 @@
+/*!
+ * \file beidou_b2ad pcps_acquisition.h
+ * \brief Adapts a PCPS acquisition block to an Acquisition Interface for
+ *  BEIDOU B2a signals
+ * \authors <ul>
+ *          <li> Sara Hrbek, 2018. sara.hrbek(at)gmail.com gsoc 2018
+ *          </ul>
+ *
+ * -------------------------------------------------------------------------
+ *
+ * Copyright (C) 2010-2018  (see AUTHORS file for a list of contributors)
+ *
+ * GNSS-SDR is a software defined Global Navigation
+ *          Satellite Systems receiver
+ *
+ * This file is part of GNSS-SDR.
+ *
+ * GNSS-SDR is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * GNSS-SDR is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GNSS-SDR. If not, see <https://www.gnu.org/licenses/>.
+ *
+ * -------------------------------------------------------------------------
+ */
+
+#ifndef GNSS_SDR_BEIDOU_B2AD_PCPS_ACQUISITION_H_
+#define GNSS_SDR_BEIDOU_B2AD_PCPS_ACQUISITION_H_
+
+#include "acquisition_interface.h"
+#include "gnss_synchro.h"
+#include "pcps_acquisition.h"
+#include "complex_byte_to_float_x2.h"
+#include <gnuradio/blocks/stream_to_vector.h>
+#include <gnuradio/blocks/float_to_complex.h>
+#include <volk_gnsssdr/volk_gnsssdr.h>
+#include <string>
+
+#include "../../../core/system_parameters/BEIDOU_B2A.h"
```

Comment:
I do not think this is the right way of adding header files. Avoid the `../../` pattern and rather add the directory to Cmakefile.txt. Look for what other files are doing. Also, did you try only using `#include "BEIDOU_B2A.h"`